lib/docker-image-tree.coffee

```diff
@@ -32,4 +33,6 @@ exports.annotateTree = annotateTree = (layer_mtimes, tree) ->
 	}
 
 exports.dockerImageTree = dockerImageTree = ->
-	docker.listImagesAsync(all: true).then(createTree)
+	dockerUtils.getDocker()
+	.then (docker) ->
```

Comment:
Yeah much nicer, I really like `Promise.call` actually (after just finding out about it :p).